plugins/inventory/nmap.py

```diff
@@ -30,10 +30,14 @@
         address:
             description: Network IP or range of IPs to scan, you can use a simple range (10.2.2.15-25) or CIDR notation.
             required: true
+            env:
+              - name: NMAP_ADDRESS
```

Comment:
The convention is to use environment variables with the prefix `ANSIBLE_`:
```suggestion
              - name: ANSIBLE_NMAP_ADDRESS
                version_added: 6.6.0
```